spec/features/lists_feature_spec.rb

```diff
@@ -185,12 +194,12 @@
       end
 
       scenario "user can update a listing's priority", js: true do
-        listing
+        create(:listing, list: list, movie: fargo)
         page.driver.browser.manage.window.resize_to(1280,800)
         sign_in_user(user)
         visit(user_list_path(user, list))
-        find("#modal_link_#{movie.tmdb_id}")
-        find("#modal_link_#{movie.tmdb_id}").click
+        find("#modal_link_#{fargo.tmdb_id}")
+        find("#modal_link_#{fargo.tmdb_id}").click
```

Comment:
Good clarification here -- especially given your last struggs with all of that hardcoded stuff leading to confusion. 😓